pkg/command/map_string.go

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Authors of Cilium
+
+package command
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/spf13/cast"
+	"github.com/spf13/viper"
+)
+
+var keyValueRegex = regexp.MustCompile(`((\w+)=(\w+,))+((\w+)=(\w+))$`)
```

Comment:
for better visualisation and checking 

https://jex.im/regulex/#!flags=&re=((%5Cw%2B)%3D(%5Cw%2B%2C))%2B((%5Cw%2B)%3D(%5Cw%2B))%24

Reply:
Should this be `((\w+)=(\w+,))*((\w+)=(\w+))$` (note: `*` instead of `+` )? 

Otherwise, this will reject single `k=v` pairs, and always require at least two pairs (`k1=v1,k2=v2`).

Reply:
Look at this the below unit test :face_exhaling: 

```
		{
			name: "valid format with one pair",
			args: args{
				str: "k1=v1,k2=v2", --> this should be only one kv pair as per description
			},
			want: true,
		},
``